scout/server/blueprints/variant/controllers.py

```diff
@@ -45,6 +45,75 @@
 LOG = logging.getLogger(__name__)
 
 
+def tx_overview(variant_obj):
+    """Prepares the content of the transcript overview to be shown on variant and general report pages.
+       Basically show transcripts that contain RefSeq or are canonical.
+
+    Args:
+        variant_obj(dict)
+    """
+    overview_txs = []  # transcripts to be shown in transcripts overview
+    for gene in variant_obj.get("genes", []):
+        for tx in gene.get("transcripts", []):
+            ovw_tx = {}
+
+            if "refseq_identifiers" not in tx and tx.get("is_canonical", False) is False:
+                continue  # collect only RefSeq or canonical transcripts
+
+            # ---- create content for the gene column -----#
+            ovw_tx["hgnc_symbol"] = (
+                gene["common"].get("hgnc_symbol", gene.get("hgnc_id"))
+                if gene.get("common")
+                else gene.get("hgnc_id")
+            )
+            ovw_tx["hgnc_id"] = gene.get("hgnc_id")
+
+            # ---- create content for the Refseq IDs column -----#
+            ovw_tx["decorated_refseq_ids"] = []
+            ovw_tx["mane"] = tx.get("mane_select_transcript", "")
+            ovw_tx["mane_plus"] = tx.get("mane_plus_clinical_transcript", "")
+
+            for refseq_id in tx.get("refseq_identifiers", []):
+                decorated_tx = None
+                if ovw_tx["mane"] and ovw_tx["mane"].startswith(refseq_id):
+                    decorated_tx = ovw_tx["mane"]
+                elif ovw_tx["mane_plus"] and ovw_tx["mane_plus"].starstwith(refseq_id):
+                    decorated_tx = ovw_tx["mane_plus"]
+                elif refseq_id.startswith("XM"):
+                    decorated_tx = "".join(
+                        ['<font class="text-muted font-italic">', refseq_id, "</font>"]
```

Comment:
How about flagging it as muted transcripts and defer the styling of it to the template, much like those flagged mane etc?

Reply:
That can certainly be done, good suggestion!